packages/react/src/components/MultiSelect/next/FilterableMultiSelect.js

```diff
@@ -0,0 +1,598 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { WarningFilled16, WarningAltFilled16 } from '@carbon/icons-react';
+import cx from 'classnames';
+import Downshift from 'downshift';
+import isEqual from 'lodash.isequal';
+import PropTypes from 'prop-types';
+import React, { useState, useRef, useEffect } from 'react';
+import { defaultFilterItems } from '../../ComboBox/tools/filter';
+import { sortingPropTypes } from '../MultiSelectPropTypes';
+import ListBox, { PropTypes as ListBoxPropTypes } from '../../ListBox';
+import { ListBoxTrigger, ListBoxSelection } from '../../ListBox/next';
+import { match, keys } from '../../../internal/keyboard';
+import Selection from '../../../internal/Selection';
+import { mapDownshiftProps } from '../../../tools/createPropAdapter';
+import { defaultItemToString } from '../tools/itemToString';
+import mergeRefs from '../../../tools/mergeRefs';
+import { useId } from '../../../internal/useId';
+import { defaultSortItems, defaultCompareItems } from '../tools/sorting';
+import { FeatureFlagContext } from '../../FeatureFlags';
+import { usePrefix } from '../../../internal/usePrefix';
+
+const FilterableMultiSelect = React.forwardRef(function FilterableMultiSelect(
+  {
+    ariaLabel,
+    className: containerClassName,
+    compareItems,
+    direction,
+    disabled,
+    downshiftProps,
+    filterItems,
+    helperText,
+    hideLabel,
+    id,
+    initialSelectedItems,
+    invalid,
+    invalidText,
+    items,
+    itemToElement,
+    itemToString,
+    light,
+    locale,
+    open,
+    onChange,
+    onMenuChange,
+    placeholder,
+    titleText,
+    type,
+    selectionFeedback,
+    size,
+    sortItems,
+    translateWithId,
+    useTitleInItem,
+    warn,
+    warnText,
+  },
+  ref
+) {
+  const [isOpen, setIsOpen] = useState(open);
+  const [prevOpen, setPrevOpen] = useState(open);
+  const [inputValue, setInputValue] = useState('');
+  const [topItems, setTopItems] = useState([]);
+  const [inputFocused, setInputFocused] = useState(false);
+  const [highlightedIndex, setHighlightedIndex] = useState(null);
+  const textInput = useRef();
+  const filterableMultiSelectInstanceId = useId();
+
+  const context = FeatureFlagContext;
+  let enabled;
+
+  if (context.enabled) {
+    enabled = context.enabled('enable-v11-release');
+  }
```

Comment:
We can update this to:

```suggestion
  const enabled = useFeatureFlag('enable-v11-release');
```